zinit-install.zsh

```diff
@@ -1456,7 +1462,13 @@ builtin source "${ZINIT[BIN_DIR]}/zinit-side.zsh" || {
     local user=$1 plugin=$2 urlpart=$3
 
     if [[ -z $urlpart ]] {
-        local url=https://github.com/$user/$plugin/releases/$ICE[ver]
+        local tag_version=${ICE[ver]}
+        if [[ -z $tag_version ]]; then
+            tag_version="$( { .zinit-download-file-stdout $site/latest || .zinit-download-file-stdout $site/latest 1; } 2>/dev/null | \
```

Comment:
```suggestion
            local releases_url=https://github.com/$user/$plugin/releases/latest
            tag_version="$( { .zinit-download-file-stdout $releases_url || .zinit-download-file-stdout $releases_url 1; } 2>/dev/null | \
```

Reply:
Thanks for working on this @yutkat! I checked out your PR locally and needed to make the above change in order for it work. It looks like `$site` isn't defined in scope at this point in the script.

Reply:
Thanks!